Library/Formula/open-mesh.rb

```diff
@@ -16,11 +16,17 @@ class OpenMesh < Formula
   head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
 
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on "qt" => :optional
```

Comment:
To be concrete, I do not know when and where the `patch` command will be called. Is `patch` applied on untarred files? under the root dir?